macos/FluentUI/AvatarView/AvatarView.swift

```diff
@@ -480,3 +531,26 @@ fileprivate extension CGPath {
 		return CGPath(ellipseIn: NSRect(x: 0, y: 0, width: diameter, height: diameter), transform: nil)
 	}
 }
+
+fileprivate extension NSColor {
```

Comment:
This eventually can be leveraged from in other parts of the internal library code. Maybe move it to the Colors.swift file?